docs/recipes/precompiling-with-webpack.md

```diff
@@ -7,20 +7,26 @@ The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) ment
 ###### webpack.config.js
 
 ```js
+const path = require('path');
+// To pre-compile folder with multiple test files, we can use glob
+// see http://stackoverflow.com/questions/32874025/how-to-add-wildcard-mapping-in-entry-of-webpack/34545812#34545812
+// const glob = require('glob');
+
 const nodeExternals = require('webpack-node-externals');
 
 module.exports = {
 	entry: ['src/tests.js'],
+	// entry: glob.sync('./test/**/*.js'),
```

Comment:
The recipe discusses precompiling source files. I'm not sure why the entry here is `src/tests.js` but it seems even more incongruent to add `test/**/*.js` entries.

How are you using this?

Perhaps the globbing approach can be mentioned in the paragraph that follows the example code, with a link to the Stack Overflow post?